hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/fs/XAttr.java

```diff
@@ -62,7 +62,8 @@
     TRUSTED((byte)1),
     SECURITY((byte)2),
     SYSTEM((byte)3),
-    RAW((byte)4);
+    RAW((byte)4),
+    PROVENANCE((byte)5);
```

Comment:
can you add a description in the above comment?

Reply:
done

Reply:
Just before the 1) at the beginning of the comment there is a sentance saying "Five namespaces are defined: user, trusted, security, system and raw." could you update it to say 6 and add yours?

Reply:
done